client/src/game/Game.js

```diff
@@ -65,6 +65,7 @@ class Game extends Phaser.Scene {
         PlayerState.setStats(playerData.stats);
         PlayerState.setTasks(playerData.tasks);
         InventoryState.setItems(data.inventory.items);
+        InventoryState.loadHotbar(data.accountID, data.isLoggedIn);
```

Comment:
I don't think both of these are needed. If an accountID is given, then it is a player joining the game with an existing account, then they by definition are logged in, so you don't need to use `isLoggedIn`, just check if `accountID` is truthy.